module/multimedia/migrations/Version20210811100500.php

```diff
@@ -0,0 +1,49 @@
+<?php
+/**
+ * Copyright © Ergonode Sp. z o.o. All rights reserved.
+ * See LICENSE.txt for license details.
+ */
+
+declare(strict_types=1);
+
+namespace Ergonode\Migration;
+
+use Doctrine\DBAL\Schema\Schema;
+use Ergonode\Multimedia\Domain\ValueObject\Hash;
+use Ergonode\Multimedia\Infrastructure\Service\Migration\FileMigrationService;
+use Ergonode\SharedKernel\Domain\Aggregate\MultimediaId;
+use Symfony\Component\DependencyInjection\ContainerAwareInterface;
+use Symfony\Component\DependencyInjection\ContainerAwareTrait;
+
+/**
+ * Auto-generated Ergonode Migration Class:
+ */
+final class Version20210811100500 extends AbstractErgonodeMigration implements ContainerAwareInterface
+{
+    use ContainerAwareTrait;
+
+    public function up(Schema $schema): void
+    {
+        /** @var FileMigrationService $fileService */
+        $fileService = $this->container
+            ->get(FileMigrationService::class);
+
+        $data = $this->getHash();
+
+
+        foreach ($data as $row) {
+            $fileService->migrateFile(new MultimediaId($row['id']), new Hash($row['hash']), $row['extension']);
```

Comment:
I think it'd be wise to catch exceptions, log the output and carry on with migration. Otherwise we might end up with migrated half of the files and just 1 skipped because of single issue